.cirrus.yml

```diff
@@ -79,6 +79,11 @@ gcp_docker_auth_win: &gcp_docker_auth_win
     - gcloud auth configure-docker --quiet %GCP_REGION%-docker.pkg.dev
 
 
+on_failure_googlecompute: &on_failure_googlecompute
```

Comment:
Maybe add a comment saying something like "Occasionally image creation succeeds despite signalling a failure. Without further action that'd result in accesses to the image failing due to lacking privileges. Deprecating the image avoids it being used by default, but leaves the image around for potential investigation."